test-build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+clear
+flatpak-builder --repo=testing-repo --force-clean build-dir com.github.xournalpp.xournalpp.yaml
+flatpak --user remote-add --if-not-exists --no-gpg-verify xpp-testing-repo testing-repo
+flatpak --user install xpp-testing-repo com.github.xournalpp.xournalpp -y
```

Comment:
there's also --install for fp-builder, so all the commands can probably collapsed into one.

Reply:
I'm fine with removing the script altogether, but for now let's keep it until we've settled the issue related to TinyTex